api/peer_counter.go

```diff
@@ -0,0 +1,43 @@
+package api
```

Comment:
I know this is dead simple but `p2p.Peers` basically implements the same maybe it's better to use it instead of having more implementations ?

Reply:
It wasn't as simple as it would seem, but I think it's a good change. 👍